vizro-core/docs/pages/user-guides/data.md

```diff
@@ -268,8 +268,11 @@ data_manager["no_expire_data"].timeout = 0
 
 ### Parametrize data loading
 
-You can supply arguments to your dynamic data loading function that can be modified from the dashboard.
-For example, if you are handling big data then you can use an argument to specify the number of entries or size of chunk of data.
+You can supply arguments to your dynamic data loading function that can be modified from the dashboard. For example:
```

Comment:
There seems to be an issue with the itemization in the rendered docs

Reply:
Good spot, fixed!